server/segments_endpoint.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"database/sql"
+	"encoding/json"
+	"net/http"
+
+	"github.com/dadikovi/race-timer/server/core"
+)
+
+type SegmentEndpoint struct{}
+
+func (se *SegmentEndpoint) create(w http.ResponseWriter, r *http.Request, db *sql.DB) {
```

Comment:
Why is `SegmentEndpoint` a type? `create` does not use `se` at all. Maybe there are future plans where it makes a sense but probably not right now.

Reply:
Yes, thats true, no point for this.